### DISCUSSION TITLE
Case sensitive search (for contents of issue comments)

### DISCUSSION THREAD

**Discussion by HighCommander4** [community/community#16165] [Projects and Issues]
It would be helpful to be able to specify that a search on the issues page (which searches the contents of issues such as comments) should be case-sensitive.

For example, in our project, "USR" is a domain-specific acronym and only a few issues reference it, but they're hard to find because searching for "USR" turns up hundreds of issues that reference paths containing `/usr/...`

**Comment by mwichmann** [comment]
Certainly agree with this - just ran across this discussion, but this has been an irritant for ages.  e.g. I have a project where there's a specific object in the API called `Variables`, and trying to find old issues that did not have the relevant label attached, but there's a flood of noise from inevitable reference to variables.
